lib/bloc/nostr/nostr_bloc.dart

```diff
@@ -39,6 +54,14 @@ class NostrBloc with AsyncActionsHandler {
     nostrPrivateKey = await _secureStorage.read(key: "nostrPrivateKey");
```

Comment:
It seems that we are storing the private key in the golang library but read it from the secure storage. Can you explain that?

Reply:
We are storing and fetching the private key in the golang library but only for the first time when a nostrBloc instance is made . Otherwise, if a new instance is made the private key is fetched from the secure storage(where it was stored the first time the key was created and fetched from the golang library).

Reply:
If it is stored in the golang library why use secure storage at all?

Reply:
okay got it, will remove the secure storage part.